docs/chapters/workflow/release/release-process.rst

```diff
@@ -11,9 +11,12 @@ licensing reasons, as there are more obligations when doing a binary release.
 
 Binary release
 --------------
-Binary images (starting from Pelux 3.0) are now available for `download`_. Each new release will have a set of binary images supporting the various variants as mentioned below.
+Binary images (starting from Pelux 3.0) are now available for `download`_.
+Each new release will have a set of binary images supporting the various
+variants as mentioned below.
 
-Currently 4 different variants are released. Every hardware offers the qtauto image as well as the minimal image, while QEMU offers only the minimal image. 
+Currently 4 different variants are released. Every hardware offers the qtauto
```

Comment:
4 different variants is incorrect.. you can take in consideration the -dev variants, or the -update ones.. even without them.. its still wrong. 2 for each hardware, makes 6 variants + qemu. I think instead of giving the explanation of the variants, you can just not mention it. 

`Every hardware offers the qtauto image as well as the minimal image, while QEMU offers only the minimal image. `